xbmc/windowing/mir/WinSystemMirGLESContext.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#ifndef WINDOW_SYSTEM_MIR_GLES_CONTEXT_H_
```

Comment:
we require it in many places. a compiler that does not support pragma won't ever be able to build kodi